test/ng/rootScopeSpec.js

```diff
@@ -1097,6 +1097,94 @@ describe('Scope', function() {
       expect(log).toEqual('');
     });
 
+    it('should remove all watchers once one-time/constant bindings are stable', function() {
+      //empty
+      scope.$watchGroup([], noop);
+      //single one-time
+      scope.$watchGroup(['::a'], noop);
+      //multi one-time
+      scope.$watchGroup(['::a', '::b'], noop);
+      //single constant
+      scope.$watchGroup(['1'], noop);
+      //multi constant
+      scope.$watchGroup(['1', '2'], noop);
+
+      expect(scope.$$watchersCount).not.toBe(0);
+      scope.$apply('a = b = 1');
+      expect(scope.$$watchersCount).toBe(0);
+    });
+
+    it('should maintain correct new/old values with one time bindings', inject(function($rootScope, $exceptionHandler) {
```

Comment:
`$exceptionHandler` is not used. And you could probably use `scope` instead of `$rootScope` to avoid `inject()` altogether.